internal/login/login_linux.go

```diff
@@ -0,0 +1,19 @@
+//go:build !windows && !darwin
+
+package login
+
+import (
+	"context"
+	"os/exec"
+)
+
+func RunOpenCmd(ctx context.Context, input string) error {
+	if err := exec.CommandContext(ctx, "xdg-open", input).Run(); err != nil {
+		if err := exec.CommandContext(ctx, "x-www-browser", input).Run(); err != nil {
+			if err := exec.CommandContext(ctx, "wslview", input).Run(); err != nil {
+				return exec.CommandContext(ctx, "sensible-browser", input).Run()
+			}
+		}
+	}
```

Comment:
```suggestion
	if f, err := os.ReadFile("/proc/sys/kernel/osrelease"); err == nil && bytes.Contains(f, []byte("WSL")) {
		return exec.CommandContext(ctx, "wslview", input).Run()
	}
	return exec.CommandContext(ctx, "xdg-open", input).Run()
	}
```

We already have a way to check if the system is WSL. I'd suggest using the same check here for consistency.

It also seems that `x-www-browser` and `sensible-browser` are generic linux scripts not specific to WSL. I've removed them to keep the dependencies simple.